test/spec/email/sendgrid.spec.ts

```diff
@@ -0,0 +1,106 @@
+require('module-alias/register');
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import { Application } from 'express';
+import {
+  EmailService,
+} from '../../../src/modules/communication';
+import { initApp } from '../../helpers/spec-helpers';
+import mail from '@sendgrid/mail';
+
+chai.use(chaiHttp);
+let expressApp: Application;
+let sinonSandbox: sinon.SinonSandbox;
+
+describe('Email Service.', function() {
```

Comment:
Specs are hard to read, there are no logical line breaks.

Ex: See this - https://github.com/sendgrid/sendgrid-nodejs/blob/78dae9ed05cc54bd50b4c0354a5e15855d83db60/packages/helpers/classes/response-error.js

You will see how there is logical chunk of separation.